proxy/postgres/src/graal/reflect-config.json

```diff
@@ -1209,10 +1001,5 @@
 {
   "name":"com.zaxxer.hikari.HikariConfig",
   "allDeclaredFields":true
-},
-{
-  "name":"com.google.protobuf.any.Any",
-  "allDeclaredMethods":true,
```

Comment:
@dwijnand this needs to stay

Reply:
It's now in `proxy/core/src/main/resources/META-INF/native-image/io.cloudstate/cloudstate-proxy-core/cloudstate-reflect-config.json.conf`.